CHANGELOG.md

```diff
@@ -7,8 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+
```

Comment:
Can you remove this additional space and squash please? I think this PR needs another rebase too as someone has just versioned the gem.